sdk-core/transactions-factories/token_management_transactions_factory.md

```diff
@@ -0,0 +1,174 @@
+## TokenManagementTransactionsFactory
+
+A class that provides methods for creating transactions for token management operations.
+
+```
+class TokenManagementTransactionsFactory:
+    createIssueFungibleTransaction({
+        sender: IAddress;
+        tokenName: string;
+        tokenTicker: string;
+        canFreeze: boolean;
+        canWipe: boolean;
+        canPause: boolean;
+        canTransferNFTCreateRole: boolean;
+        canChangeOwner: boolean;
+        canUpgrade: boolean;
+        canAddSpecialRoles: boolean;
+
+        // Optionals:
+        transactionNonce?: INonce;
+        value?: ITransactionValue;
+        gasPrice?: IGasPrice;
+        gasLimit?: IGasLimit;
+        guardian? : IAddress;
+    }): Transaction;
+
+    createIssueSemiFungibleTransaction({
```

Comment:
:+1: for having separate issue functions, they will bring clarity. Maybe we can split this large class in 3 (for the fungible, non-fungible and semi-fungible / meta esdts)
All 3 classes can derive the same "base" class with common functionality found on all ESDTs

Reply:
To be discussed with NFT service & DEX backend teams.